### DISCUSSION TITLE
Setting RenderOptions Programmatically

### DISCUSSION THREAD

**Discussion by najak3d** [AvaloniaUI/Avalonia#7652] [Q&A] [answered]
First off, eventually, might be nice to have a Markup.Declarative discussion location, because that's what we're using, which differs significantly from XAML approach:

We are replacing this Window XAML: 

      RenderOptions.BitmapInterpolationMode="HighQuality"

With this code:

            Avalonia.Media.RenderOptions.SetBitmapInterpolationMode(myWindow, BitmapInterpolationMode.HighQuality);

Is this the best way to replace XAML with C#?  Just want to make sure that we're doing this the "best way" to avoid developing bad habits.

**Comment by kekekeks** [comment, marked as answer]
That's exactly what XAML compiler generates, yes.
